src/Compilers/CSharp/Test/Emit/Microsoft.CodeAnalysis.CSharp.Emit.UnitTests.csproj

```diff
@@ -24,6 +24,7 @@
   <ItemGroup>
     <PackageReference Include="Microsoft.CSharp" Version="$(MicrosoftCSharpVersion)" />
     <PackageReference Include="Microsoft.DiaSymReader" Version="$(MicrosoftDiaSymReaderVersion)" />
+    <PackageReference Include="Basic.Reference.Assemblies.Net50" Version="$(BasicReferenceAssembliesNet50Version)" />
```

Comment:
nit: Would it be useful to shuffle some tests around, so that all tests that depend on Net50 live together in a single project?

Reply:
The big win here was removing the ref from the entire IDE layer which was a big pay for play win. I think the usage in the compiler is just fine. This change was really motivated by the idea of letting us keep writing the tests the way we like but reduce the impact of doing so.